src/modules/peek/Peek.FilePreviewer/Controls/BrowserControl.xaml.cs

```diff
@@ -27,6 +27,9 @@ public sealed partial class BrowserControl : UserControl, IDisposable
 
         public event DOMContentLoadedHandler? DOMContentLoaded;
 
+        private string previewBrowserUserDataFolder = System.Environment.GetEnvironmentVariable("USERPROFILE") +
+                        "\\AppData\\LocalLow\\Microsoft\\PowerToys\\Peek-Temp";
```

Comment:
Can we change the path to something like `...\PowerToys\Peek\WebView2-Temp`? Shouldn't we exclude all WV2 temp dirs from settings backup in case they contain files that are saved by settings backup?